### PR TITLE
Replace try/catch with catch()

### LINKED ISSUE #68: Replace some try/catch blocks with .catch()
There are quite a few places in the code where try/catch blocks with `let` variables just outside the try statement can be replaced with the cleaner `.catch()` method for handling errors.

### PR DESCRIPTION
**Pull request by mattbho**
# Fixes #68

## Description of changes
Replaces try/catch blocks with catch()
## Checklist

If your change touches anything under src or the README.md file
these items must be done:

- [x] CHANGELOG.md updated
- [x] `npm version` run

### CODE REVIEW

**Comment by mattbho** [comment]
Ran `npm version --unsafe-perm minor` but got a Git tagging error, saying tag v1.2.0 already exists. Please let me know how I can fix that, and once I do, this PR is ready to go.

**Comment by danecreekphotography** [comment]
Just manually edit package.json and change the version to 1.2.1 and make sure changelog.md has same.

Don't worry about running the npm version command, I'll get rid of that from the guidelines. Too problematic!

**Comment by danecreekphotography** [comment]
How was the dev container experience, besides the npm version fun?

**Comment by mattbho** [comment]
> How was the dev container experience, besides the npm version fun?

Good! I had no other problems

File: src/DeepStack.ts [hunk 1]
```diff
@@ -14,14 +14,14 @@ export default async function analyzeImage(fileName: string): Promise<IDeepStack
   const imageStream = fs.createReadStream(fileName);
   const form = { image: imageStream };
 
-  try {
-    const rawResponse = await request.post({
+  const rawResponse = await request
```
Comment (danecreekphotography) [inline src/DeepStack.ts:17]: This makes me irrationally happy.

File: package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-deepstackai-trigger",
-  "version": "1.1.5",
+  "version": "1.2.0",
```
Comment (danecreekphotography) [inline package-lock.json:3]: This needs to be 1.2.1, and now I understand why npm version exists.

Comment (danecreekphotography) [inline package-lock.json:3]:
```suggestion
  "version": "1.2.1",
```

File: src/DeepStack.ts [hunk 3]
```diff
@@ -14,14 +14,14 @@ export default async function analyzeImage(fileName: string): Promise<IDeepStack
...
+    .catch(e => {
+      log.error("DeepStack", `Failed to call DeepStack at ${process.env.DEEPSTACK_URI}: ${e.error}`);
+      return undefined;
     });
-    return JSONC.parse(rawResponse) as IDeepStackResponse;
-  } catch (e) {
-    log.error("DeepStack", `Failed to call DeepStack at ${process.env.DEEPSTACK_URI}: ${e.error}`);
-    return undefined;
-  }
+  return JSONC.parse(rawResponse) as IDeepStackResponse;
```
Comment (danecreekphotography) [inline src/DeepStack.ts:26]:
```suggestion

  return JSONC.parse(rawResponse) as IDeepStackResponse;
```

File: src/Trigger.ts [hunk 4]
```diff
@@ -236,12 +236,10 @@ export default class Trigger {
...
-    try {
-      await this._watcher.close();
-      log.info(`Trigger ${this.name}`, `Stopped listening for new images in ${this.watchPattern}`);
-    } catch (e) {
+    await this._watcher.close().catch(e => {
       log.error(`Trigger ${this.name}`, `Unable to stop watching for images: ${e}`);
       throw e;
-    }
+    });
+    log.info(`Trigger ${this.name}`, `Stopped listening for new images in ${this.watchPattern}`);
```
Comment (danecreekphotography) [inline src/Trigger.ts:243]:
```suggestion

    log.info(`Trigger ${this.name}`, `Stopped listening for new images in ${this.watchPattern}`);
```

**Pull request merged**